File: Add Code Here/JAVA/fibonacci.java
public class fibonacci{
    public static void main(String[] args) {
         
        int limit = 20;
        long[]series = new long[limit];
        series[0]=0;
        series[1]=1;
        for (int i=2;i<limit;i++){
            series[i]=series[i-1]+series[i-2];
        }
        System.out.println("Fiboonaci series upto" + limit);
         for(int i=0;i<limit;i++) {
            System.out.print(series[i] + "");
    }
}}

File: Add Code Here/Javascript/Fibonacci.js
function fibonacciRecursive(n) {
  if (n <= 0) return [];
  if (n === 1) return [0];
  if (n === 2) return [0, 1];

  const series = fibonacciRecursive(n - 1);
  series.push(series[series.length - 1] + series[series.length - 2]);
  return series;
}

const n = 10; // Change this value to the desired length of the Fibonacci series
const fibonacciSeries = fibonacciRecursive(n);
console.log(fibonacciSeries);
